src/Directory.Build.props

```diff
@@ -6,6 +6,7 @@
     <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
     <EnableSourceLink Condition=" '$(NCrunch)' == '1' ">false</EnableSourceLink>
     <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
+    <NoWarn>$NoWarn;MSB3026</NoWarn>
```

Comment:
Is `$NoWarn` supposed to be `$(NoWarn)` to allow other props to add warnings together? Or is this a special syntax for `NoWarn` that does something cool?

Reply:
No, that's a typo. :)